### PR TITLE
Add question 7: can you offer care?

### PR DESCRIPTION
**Pull request by huwd**
https://trello.com/c/xIUCjuXs/28-implement-question-7

### CODE REVIEW

**Review by bilbof — APPROVED** [review]
Excellent 👍

**Comment by bilbof** [comment]
You have a conflict (don't we all), and the test is failing because you'll need to add a route for the new path you've defined in config/routes.rb (controller not needed yet):

```
       No route matches {:action=>"show", :controller=>"coronavirus_form/offer_community_support"}
```

**Comment by huwd** [comment]
thanks @bilbof , just updated.

I've got a route (for previous) that'll depend on Issy's PR here: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/29

Guess I'll just wait until that's merged then rebase?

File: app/views/coronavirus_form/offer_care.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <meta name="description" content="<%= t('coronavirus_form.offer_care.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "coronavirus_form/validation_error" %>
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-hotel-rooms",
```
Comment (theseanything) [inline app/views/coronavirus_form/offer_care.html.erb]: Is this correct?

Comment (huwd) [reply app/views/coronavirus_form/offer_care.html.erb]: Nope... fixing, sorry copy pasta

File: config/locales/en.yml [hunk 2]
```diff
@@ -130,6 +130,15 @@ en:
...
       custom_select_error: Select which services you can provide
+    offer_care:
+      title: Can you offer social care or childcare?
+      hint: For example, nursing or assistance to other people.
+      options:
+        option_yes:
+          label: "Yes"
+        option_no:
+          label: "No"
+      custom_select_error: Select which services you can provide
```
Comment (theseanything) [inline config/locales/en.yml]: Does this make sense as a boolean form?

**Comment by huwd** [comment]
> You have a conflict (don't we all), and the test is failing because you'll need to add a route for the new path you've defined in config/routes.rb (controller not needed yet):
> 
> ```
>        No route matches {:action=>"show", :controller=>"coronavirus_form/offer_community_support"}
> ```

So i just took out that code and sent it to Sean,
He can then copy paste in over my work to make sure the test + controller redirect for next page is there.

Right now i just removed the test and made it go to thank you!

File: config/locales/en.yml (hunk 2 above)
Comment (huwd) [reply config/locales/en.yml]: Good question, will put out a shout in slack for error message copy:
Just stole this one for consistency for now: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/config/locales/en.yml#L132

**Review by theseanything — APPROVED** [review]

File: config/locales/en.yml (hunk 2 above)
Comment (huwd) [reply config/locales/en.yml]: Asked here: https://gds.slack.com/archives/C010FS40HHD/p1584788365118600

File: config/routes.rb [hunk 3]
```diff
@@ -47,6 +47,9 @@
   get "/coronavirus-form/what-kind-of-transport" => "coronavirus_form/transport_type#show"
   post "/coronavirus-form/what-kind-of-transport" => "coronavirus_form/transport_type#submit"
 
+  get "/coronavirus-form/offer-advice" => "coronavirus_form/offer_advice#show"
+  post "/coronavirus-form/offer-advice" => "coronavirus_form/offer_advice#submit"
```
Comment (bilbof) [inline config/routes.rb]: I think these routes aren't necessary, since we have `expert_advice`

Comment (huwd) [reply config/routes.rb]: ah great spot, i added those for the back button, i guessed the name wrong!
Removed.

**Pull request merged**